include/bsp/iotc_bsp_crypto.h

```diff
@@ -29,115 +29,96 @@ extern "C" {
 
 /**
  * @file  iotc_bsp_crypto.h
- * @brief Encode and sign JWT credentials.
- *
- * These functions implement a cryptography library to: generate and sign JWTs
- * via ECC keys and SHA256; or format signed JWTs as URL-safe base64 strings.
- * Reference implementations are in the
- * <a href="~/src/bsp/crypto"><code>src/bsp/crypto</code></a> directory.
- *
- * The BSP uses the private key data or slot number provided to the
- * <code>iotc_connect()</code> function. The BSP doesn't format the JWT itself;
- * the BSP only signs JWTs. The <a href="../../api/html/index.html">Device SDK
- * API</a> formats the original JWT.
- *
- * All cryptography functions issue an <code>iotc_bsp_crypto_state_e</code>
- * status message.
+ * @brief Implements a TLS library for creating JSON Web Tokens.
```

Comment:
Unfortunately this isn't correct so. The previous expression is more accurate: "implement a crypto lib...".
TLS libraries are 3rd party libs: mbedTLS and wolfSSL. This SDK does not implement TLS lib.

I would phrase somehow "These function implement a small crypto lib to generate JWT. (ECC, SHA256 and base64)..."
Please use the small cryptography library instead of TLS library. Thx!

Reply:
Done